src/hotspot/cpu/x86/gc/z/zBarrierSetAssembler_x86.cpp

```diff
@@ -146,10 +146,33 @@ void ZBarrierSetAssembler::load_at(MacroAssembler* masm,
   __ movdqu(Address(rsp, xmm_size * 1), xmm1);
   __ movdqu(Address(rsp, xmm_size * 0), xmm0);
 
+  const int opmask_size = 8;
```

Comment:
The code in `ZBarrierSetAssembler::load_at` doesn't handle vector registers. Why should it care about predicate registers?